proto/task_service.proto

```diff
@@ -35,6 +35,10 @@ message CreateTaskRequest {
   batch_plan.TaskId task_id = 1;
   batch_plan.PlanFragment plan = 2;
   uint64 epoch = 3;
+  // The partition to read for scan tasks.
+  //
+  // For tasks without a scan node, this field is ignored.
+  repeated common.RangeInclusive vnode_ranges = 4;
```

Comment:
I'm still not convinced why `vnode_ranges` should be task rather than plan node?